src/containers/wordpress/wordpressPage.js

```diff
@@ -47,8 +48,9 @@ class WordpressPage extends Component {
     return (
       <div className={style.wordPressContainer}>
         <HeadMetaTags title={title} />
+        {slug === 'home' && <Carousel /> }
```

Comment:
Since it looks like the home page is now entirely disconnected from wordpress it seems like it would be better to handle this by changing the react-router configuration instead of adding logic here.